### PR TITLE
Feature/tra 279 Create a cron job for sending quarterly reminder

### PR DESCRIPTION
**Pull request by Wojciech-Florczak**
Create a cron job for sending quarterly reminder

https://apptension.atlassian.net/browse/TRA-279

image

### CODE REVIEW

File: functions/src/constants/slack.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    text: {
+      type: 'mrkdwn',
+      text: "Hey there! :wave: We want to encourage everyone to develop as a company, not just as individuals. Make sure to check your skills matrix and update it with any new skills you've acquired!",
+    },
+  },
+  {
+    type: 'section',
+    text: {
+      type: 'mrkdwn',
+      text: `You can do it <https://radar.apptension.com/matrix/knowledge|here>`,
```
Comment (Balldoro) [inline functions/src/constants/slack.ts:16]: Just a small note, I would remove that `/knowledge` from link 😄

File: functions/src/cron-jobs/slack.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+  const usersIdsList = filteredList.map((user) => user.id);
+
+  return usersIdsList as string[];
+};
+
+const sendMessageToUsers = async () => {
+  const { SLACK_TOKEN } = getSecrets();
+  const slack = getSlackClient(SLACK_TOKEN);
+  const userIds = await getSlackUsersIds();
```
Comment (Balldoro) [inline functions/src/cron-jobs/slack.ts:32]: Maybe I'm wrong but I think this function also should have its own `try/catch`

**Review by Balldoro — CHANGES_REQUESTED** [review]

File: functions/src/cron-jobs/slack.ts (hunk 2 above)
Comment (Wojciech-Florczak) [reply functions/src/cron-jobs/slack.ts:35]: I think this combination will go well 
```js
if (usersList.error) console.error('Failed to fetch slack users data');
if (!usersList.members) return [];
```
WDYT?

File: functions/src/constants/slack.ts (hunk 1 above)
Comment (Wojciech-Florczak) [reply functions/src/constants/slack.ts]: done ✅

File: functions/src/cron-jobs/slack.ts (hunk 2 above)
Comment (Balldoro) [reply functions/src/cron-jobs/slack.ts:35]: yup, should be good enogh

**Review by Balldoro — APPROVED** [review]

**Pull request merged**